xml/MAIN.VMDP.xml

```diff
@@ -54,7 +54,40 @@
     to existing operating systems.
    </para>
   </sect2>
-
+  <sect2 xml:id="sec-vmdp-signature">
+   <title>VMDP signature</title>
+   <sect3 xml:id="sec-vmdp-signature-vmdp-win">
+    <title>VMDP-WIN-VERSION</title>
+    <para>
+     VMDP-WIN-VERSION contains the Microsoft WHQL signed sys and cat files for the
+     Windows Server versions of the drivers contained in vmdp-211015.  The Windows
+     Server drivers will also run in the corresponding Windows client versions.
+     Windows 10 32 bit client drivers are SUSE signed and Microsoft attestation
+     signed. All other 32 bit client drivers are SUSE signed only.
+    </para>
+    <para>
+     The 64 bit SUSE signed only drivers can only be installed after the initial
+     Windows installation. Windows must set Testsigning mode on for the drivers
+     to load on reboot.
+    </para>
+   </sect3>
+   <sect3 xml:id="sec-vmdp-signature-vmdp-win-community">
+    <title>VMDP Community</title>
+    <para>
+     VMDP-WIN-VERSION-Community contains the same drivers as found in vmdp-211015.
```

Comment:
What is VMDP-WIN-VERSION-Community? A package or a file?

Reply:
Its a package available for the community, a built package available without SUSE support